gptstonks_api/main.py

```diff
@@ -161,8 +196,61 @@ async def run_model_in_background(job_id: str, query: str):
 
 @app.get("/get_processing_result/{job_id}")
 async def get_processing_result(job_id: str):
+    """
+    Endpoint to fetch the processing result using a job ID.
+
+    Args:
+        job_id (str): Unique identifier for the job.
+
+    Returns:
+        dict: Contains either the status of processing or the final result once completed.
+    """
     result = results_store.get(job_id)
     if result is None:
         return {"status": "processing"}
     else:
         return {"status": "completed", "result": result}
+
+
+@app.get("/get_api_keys")
+async def get_api_keys():
+    """
+    Endpoint to retrieve the current API keys information.
+
+    Args:
+        None
+
+    Returns:
+        dict: Contains information about the current API keys.
+    """
+
+    keys = openbb.keys.get_keys_info()
+    return {"result": keys}
+
+
+@app.post("/set_api_keys")
+async def set_api_keys(request: Request):
+    """
+    Endpoint to set new API keys.
+    The new keys are set in the OpenBB SDK and also stored in a JSON file.
+
+    Args:
+        request (Request): FastAPI request object containing the keys to set.
+
+    Returns:
+        dict: Status message confirming if keys were set correctly or not.
+    """
+
+    data = await request.json()
+    keys_list = data.get("keys_dict")
+    persist = data.get("persist") or True
+    show_output = data.get("show_output") or False
```

Comment:
I believe you can just do `data.get("show_output", False)`. Same in the previous ones.